hudi-examples/src/main/java/org/apache/hudi/examples/common/RandomJsonSource.java

```diff
@@ -45,4 +45,8 @@ public RandomJsonSource(TypedProperties props, JavaSparkContext sparkContext, Sp
 
     return new InputBatch<>(Option.of(sparkContext.parallelize(inserts, 1)), commitTime);
   }
+
+  @Override
+  public void onCommit(String lastCkptStr) {
```

Comment:
for these empty `onCommit` method, would we define a default method `onCommit(String lastCkpStr)`?

Reply:
Yes, I thought about it first but Source being an abstract class dropped that idea, should we add a new interface like `SourceCallback` which will contain this default method ?

Reply:
add a new interface sounds good.

Reply:
done, as discussed added `SourceCommitCallback` interface with default method.